### PR TITLE
Add QoS page

### PR DESCRIPTION
**Pull request by kevincela**
- Add QoS configuration page
- Generalize zone icon coloring for reuse in other components

Card: https://trello.com/c/P8EzabNk/174-qos-configuration

### CODE REVIEW

File: public/i18n/en/translation.json [hunk 1]
```diff
@@ -920,6 +927,24 @@
...
+      "download_speed": "Download speed",
+      "download_speed_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
+      "upload_speed": "Upload speed",
+      "upload_speed_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
+      "status": "Status",
+      "delete_qos_interface": "Delete QoS interface",
+      "delete_qos_interface_message": "Are you sure you want to delete '{interface}'?",
+      "choose_interface": "Choose interface",
+      "add_interface": "Add interface"
+    },
```
Comment (Tbaile) [inline public/i18n/en/translation.json:947]: @cotosso, @gsanchietti, any lorem ipsum replacement?

**Review by Tbaile — APPROVED** [review]

File: public/i18n/en/translation.json (hunk 1 above)
Comment (gsanchietti) [reply public/i18n/en/translation.json:947]: On it!

File: public/i18n/en/translation.json [hunk 2]
```diff
@@ -920,6 +927,24 @@
...
+    "qos": {
+      "title_short": "QoS",
+      "title": "Quality of Service",
+      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
+      "no_interface_found": "No interface found",
+      "add_qos_interface": "Add QoS interface",
+      "edit_qos_interface": "Edit QoS interface",
+      "interface": "Interface",
+      "download_speed": "Download speed",
+      "download_speed_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:939]:
```suggestion
      "download_speed_tooltip": "Enter the incoming bandwidth in Mbit. It is recommended to slightly reduce the measurement by approximately 5-10%.",
```

File: public/i18n/en/translation.json [hunk 3]
```diff
@@ -920,6 +927,24 @@
...
+      "upload_speed": "Upload speed",
+      "upload_speed_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:941]:
```suggestion
      "upload_speed_tooltip": "Enter the outgoing bandwidth in Mbit. It is recommended to slightly decrease the measurement by approximately 5-10%.",
```

File: public/i18n/en/translation.json (hunk 1 above)
Comment (gsanchietti) [reply public/i18n/en/translation.json:947]: Done. @filippocarletti can you check please?

File: public/i18n/en/translation.json [hunk 4]
```diff
@@ -920,6 +927,24 @@
       "destination_url_tooltip": "This path specifies the location to which incoming requests will be forwarded. Example: 'http://destination-server:port/path'",
       "allowed_networks_tooltip": "Access permitted only from the following IPv4/IPV6 networks"
     },
+    "qos": {
+      "title_short": "QoS",
+      "title": "Quality of Service",
+      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
```
[the remark this answers is not in the document]
Comment (cotosso) [reply public/i18n/en/translation.json:933]:
```suggestion
      "description":"The Quality of Service (QoS) ensures fair distribution of bandwidth among network connections, dynamically adjusts rate limits based on network conditions, and addresses bufferbloat issues. 
Configure each network interface that you want QoS managed (usually only WANs are managed).
Make sure to configure the input and output bandwidth accordingly to what is actually available to allow the algorithm to operate at its best."
```

File: public/i18n/en/translation.json (hunk 1 above)
Comment (cotosso) [reply public/i18n/en/translation.json:947]: I just added my suggestion

File: public/i18n/en/translation.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (kevincela) [reply public/i18n/en/translation.json]: I've used @cotosso's version of the description, let me know if that's ok with you!

**Pull request merged**